ItemDesc.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+# coding=utf8
+import codecs
+import fnmatch
+import json
+import os
+import sys
+from collections import OrderedDict
+
+if len(sys.argv) < 2:
+    sys.exit(os.EX_NOINPUT)
+
+dir = sys.argv[1]
+
+FS3 = dict()
+FS4 = dict()
+
+explain_files = [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_*.txt')
+]
+
+explain_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Explain_Actor_*.txt')
+]
+
+explain_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Explain_SkillRing.txt')
+]
+
+explain_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Explain_System.txt')
+]
+
+explain_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Items_Leftovers.txt')
+]
+
+explain4_files = [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_Stack_Gat*.txt')
+]
+
+explain4_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_Stack_Orderitem.txt')
+]
+
+explain4_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_QuestTrigger.txt')
+]
+
+explain4_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_Stack_RingL.txt')
+]
+
+explain4_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_Stack_Music.txt')
+]
+
+explain4_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_Stack_RingR.txt')
+]
+
+explain4_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_AvatarWPN_*.txt')
+]
+
+explain4_files += [
+    os.path.join(dirpath, f)
+    for dirpath, dirnames, files in os.walk(dir)
+    for f in fnmatch.filter(files, 'Item_AvatarWPN_*.txt')
+]
+
+explain3_files = [x for x in explain_files if x not in explain4_files]
```

Comment:
so, just `Item_Stack_Gat*.txt`, `Item_Stack_Orderitem.txt`, `Item_QuestTrigger.txt`, `Item_Stack_RingL.txt`, `Item_Stack_RingR.txt`, `Item_Stack_Music.txt` and `Item_AvatarWPN_*.txt` can have 4 lines?